src/vma/proto/dst_entry.h

```diff
@@ -133,8 +133,9 @@ class dst_entry : public cache_observer, public tostr, public neigh_observer
 
 	vma_ibv_send_wr* 	m_p_send_wqe;
 	uint32_t 		m_max_inline;
-	ring_user_id_t 	m_id;
+	ring_user_id_t		m_id;
 	size_t			m_max_ip_payload_size;
```

Comment:
consider using uint_16 for both parameters

Reply:
This will cause me to change type in several places. for example :

proto/dst_entry_udp.cpp:215:85: error: no matching function for call to 'min(uint16_t&, size_t)'
   size_t sz_ip_frag = min(m_max_ip_payload_size, (sz_udp_payload - n_ip_frag_offset));

I dont think it worth it..